modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -446,6 +446,34 @@ Generates a diagnostics report at `rtf-report.tar.gz`
 rtfctl report --namespace <rtf_namespace>
 ----
 
+Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The  The default value is 10.
+
+----
+rtfctl report --workers 10
+----
+
+Generate rtfctl report compression faster by using the `use-fast-compression` flag that enables the compression of the result report archive faster in exchange to have a large report file size. The default value is false.
+
+----
+rtfctl report --use-fast-compression=true
+----
+
+Select which application namespaces you want to include in your report besides from the global rtf installation namespace by using the `<rtf_namespace>`. Using this flag enables you choose what to include in your reports, and returns a faster report. 
```

Comment:
- Should "besides" be removed in the first sentence?
- Add "to" between "you choose" in the second sentence.